### DISCUSSION TITLE
ShredOS fails to boot from iventoy PXE server

### DISCUSSION THREAD

**Discussion by VKayaVW** [PartialVolume/shredos.x86_64#265] [Q&A]
Hello, i am running an iventoy boot server. (https://www.iventoy.com/en/index.html)
I get this error message:
![IMG_0272](https://github.com/user-attachments/assets/4c95915d-ac82-4869-8d35-027378b3268a)

Can somebody tell me what to do or is it even possible to boot that image with iventoy?
I am pretty new in pxe booting :D

**Comment by PartialVolume** [comment]
I'm guessing you copied the ShredOS .iso file. That doesn't include UEFI.

You may want to try the .img instead of the .iso file. The .img does include bootx64.efi

Ventoy supports .img so I would imagine that iventoy does too.

Having said that iventoy supports legacy bios so I don't know why it's complaining about a missing .EFI file. I would have thought it would then default to legacy.

**Reply by PartialVolume** [reply]
Assuming the bios supports legacy. If it doesn't then try the .img instead.

**Reply by VKayaVW** [reply]
Thanks for the reply. Unfortunately the .img file does not show up in the bootmenu like .iso

**Reply by VKayaVW** [reply]
Is there a possibility to repack the .img as an bootable iso including the bootx64.efi file?

**Reply by PartialVolume** [reply]
I'll look into whether I can add UEFI to the .iso so it boots legacy and UEFI.

**Comment by PartialVolume** [comment]
Iventoy looks interesting, I think I'll try it out when I can find a bit of spare time.
